### PR TITLE
method name enhancement

### PR DESCRIPTION
**Pull request by sensorario**
| Q             | A
| ------------- | ---
| Branch?       | 2.2
| Bug/Hotfix?   | yes
| Refactoring?  | yes
| New feature?  | no
| BC breaks?    |no
| Deprecations? | yes
| Tests pass?   | yes

This PR add new tests for `orFilters` and makes more explicit `andFilters`

### CODE REVIEW

File: src/Mado/QueryBundle/Queries/QueryBuilderFactory.php [hunk 1]
```diff
@@ -182,20 +188,22 @@ public function filter()
             );
         }
 
-        foreach ($this->filtering as $filter => $value) {
-            $this->applyFilterAnd(
-                Objects\FilterObject::fromRawFilter($filter),
-                $value,
-                Objects\Value::fromFilter($value)
-            );
+        if ($this->andFilters !== null) {
```
Comment (AlessandroMinoccheri) [inline src/Mado/QueryBundle/Queries/QueryBuilderFactory.php]: please use yoda condition

File: src/Mado/QueryBundle/Queries/QueryBuilderFactory.php [hunk 2]
```diff
@@ -90,12 +98,12 @@ public function setSorting(array $sorting = [])
 
     public function getFilters()
```
Comment (AlessandroMinoccheri) [inline src/Mado/QueryBundle/Queries/QueryBuilderFactory.php]: maybe is better to call this function getAndFilters, because we have also getOrFilters

Comment (sensorario) [reply src/Mado/QueryBundle/Queries/QueryBuilderFactory.php]: good catch!!

**Comment by AlessandroMinoccheri** [comment]
LGTM 👍

**Pull request merged**